inference-engine/src/inference_engine/generic_ie.cpp

```diff
@@ -152,9 +151,6 @@ void ngraph::op::GenericIE::validate_and_infer_types() {
             set_output_type(output_index, get_precision(output_index), Shape(outputs[output_index].dims));
         }
         initialized++;
-    } else if (reshape) {
```

Comment:
Why did you remove this exception?

Reply:
I am testing how to fix the bug,  but from your comment I guess that is not a desired solution